source/api/commands/type.md

```diff
@@ -269,7 +288,7 @@ cy.get('input[type=text]').type('Test all the things', { force: true })
 
 If the element is currently not in focus, before issuing any keystrokes Cypress will first issue a {% url `.click()` click %} to the element to bring it into focus.
 
-All of the normal events documented on {% url `.click()` click %} will fire.
+All of {% url 'the normal events' click#Events %} documented on {% url `.click()` click %} will fire.
```

Comment:
Linking through to the events that fire on `.click()` on the rationale of being helpful.